### ISSUE TITLE
Fix the dynamic stability prediction

### ISSUE THREAD

**Issue by irmakguzey** [irmakguzey/SPWGNN#6]
Right now, after the PR #5 a dynamic prediction is being made through the simulation. 
But predictions are really really close to 0 all the time. But they do also change through the iterations. I don't know the reason behind this, the data might be overfitting. But I need to look into that.

**Comment by irmakguzey** [comment]
This is fixed with the PR #7. Turns out that the problem was that calculate_stability function (the function that calculates the output from saved data for using in training) was wrong. After fixing that and changing variables such as learning rate, error function.. etc. I achieved to have predictions that have accuracy of %80.
